samples/amlv2_pid_symbol_detection_train/.vccode/launch.json

```
@@ -0,0 +1,177 @@
+                "${env:STRATIFIED_SPLIT_N_FOLD}",
+                "--mandatory-train-filenames",
+                "${env:MANDATORY_TRAIN_FILENAMES}",
+                "--mandatory-val-filenames",
+                "${env:MANDATORY_VAL_FILENAMES}",
+                "--train-output-path",
+                "train",
+                "--val-output-path",
+                "val"
+            ]
+        },
+        {
+            "name": "Python: Registering data asset",
+            "type": "python",
+            "request": "launch",
+            "module": "pipeline_steps.register_data_asset_src.main",
+            "args": [
+                "--train-input-path",
+                "sample_artifacts/register_data_asset/input/annotations.jsonl",
+                "--val-input-path",
+                "sample_artifacts/register_data_asset/input/annotations.jsonl",
+                "--output-path",
+                "--use-stratified-split",
+                "${env:USE_STRATIFIED_SPLIT}",
+                "sample_artifacts/register_data_asset/output/",
+                "--data-store-name",
+                "workspaceblobstore"
+            ],
+            "justMyCode": true,
+            "cwd": "${workspaceFolder}/src/app"
+        },
+        {
+            "name": "Python: Pipeline Step - Tag Model",
+            "type": "python",
+            "request": "launch",
+            "module": "pipeline_steps.tag_src.main",
+            "justMyCode": true,
+            "cwd": "${workspaceFolder}/src/app",
+            "args": [
+                "--model-metadata-path",
+                "sample_artifacts/tag",
+                "--model-base-name",
+                "pid_object_detection_model",
+                "--best-model-metadata-path",
+                "sample_artifacts/tag"
+            ]
+        },
+        {
+            "name": "Python: Pipeline Step - Register Model",
+            "type": "python",
+            "request": "launch",
+            "module": "pipeline_steps.register_src.main",
+            "justMyCode": true,
+            "cwd": "${workspaceFolder}/src/app",
+            "args": [
+                "--model-input-path",
+                "sample_artifacts/register",
+                "--model-base-name",
+                "pid_object_detection_model",
+                "--build-source",
+                "local",
+                "--build-id",
+                "local-build-1",
+                "--model-metadata-path",
+                "sample_artifacts/register"
+            ]
+        },
+        {
+            "name": "Python: Local Endpoint Deployment",
+            "type": "python",
+            "request": "launch",
+            "module": "automl_pipeline.deploy.main",
+            "justMyCode": true,
+            "cwd": "${workspaceFolder}/src/app",
+            "args": [
+                "--is-local-deployment",
+                "--environment-image",
+                "apmpnidemlreg.azurecr.io/inference-container-cpu:1.0.0"
```

Rename this Docker container registry to something generic:

```suggestion
                "piddetection.azurecr.io/inference-container-cpu:1.0.0"
```